### PR TITLE
Enable super dist for openmpi 

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
@conda-forge-admin restart ci

**Comment by jan-janssen** [comment]
@dalcinl  Is it possible to merge this pull request as well?

**Comment by dalcinl** [comment]
@jan-janssen Sorry, I'm on vacation, so I'm being a bit slow.

File: recipe/build.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ if [[ $mpi == "openmpi" ]]; then
   export OMPI_MCA_plm=isolated
   export OMPI_MCA_rmaps_base_oversubscribe=yes
   export OMPI_MCA_btl_vader_single_copy_mechanism=none
-  export WITH_SUPERLU_DIST=0
+  export WITH_SUPERLU_DIST=1
```
Comment (dalcinl) [inline recipe/build.sh:34]: Perhaps we should just get rid of this `WITH_SUPERLU_DIST` variable? After all, now we build with SuperLU_Dist for both MPICH and Open MPI. Could you please update the recipe accordingly?

Comment (jan-janssen) [reply recipe/build.sh]: I removed the environment variable `WITH_SUPERLU_DIST`

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
#131 is only for mpich - not for openmpi. But I am fine with merging #131 first and this one afterwards.

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by dalcinl** [comment]
@jan-janssen If this PR is a superset of #131, then let's merge this one and move on. Please confirm.

**Comment by jan-janssen** [comment]
@dalcinl Yes, it is. Merging this one would be my preferred choice.

**Pull request merged**